### PR TITLE
Add keyboard kafka with 68 key

### PR DESCRIPTION
**Pull request by LXF-YZP**
## Description
Add keyboard kafka with 68 key

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/meetlab/kafka68/matrix_diagram.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Matrix Diagram for Best
```
Comment (waffle87) [inline keyboards/meetlab/kafka68/matrix_diagram.md:1]: typo?
```suggestion
# Matrix Diagram for kafka68
```

File: keyboards/meetlab/kafka68/readme.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# kafka68 - PCB
+
+![kafka68](https://imgur.com/5w8fESLh.jpg)
```
Comment (waffle87) [inline keyboards/meetlab/kafka68/readme.md:3]:
```suggestion
![kafka68](https://i.imgur.com/5w8fESLh.jpg)
```

File: keyboards/meetlab/kafka68/info.json [hunk 3]
```diff
@@ -0,0 +1,101 @@
+{
+    "manufacturer": "lucky_studio",
+    "keyboard_name": "kafka68",
+    "maintainer": "yuezp",
+    "development_board": "bluepill",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
```
Comment (waffle87) [inline keyboards/meetlab/kafka68/info.json:9]: This is generally preferred to be enabled if possible.
```suggestion
        "extrakey": true,
        "mousekey": true,
```

File: keyboards/meetlab/kafka68/matrix_diagram.md (hunk 1 above)
Comment (LXF-YZP) [reply keyboards/meetlab/kafka68/matrix_diagram.md]: > typo?

Sorry, I'll change it

File: keyboards/meetlab/kafka68/info.json (hunk 3 above)
Comment (LXF-YZP) [reply keyboards/meetlab/kafka68/info.json:9]: thank you！

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**